recipes/twitch-libsoundtrackutil/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+class TwitchLibSoundtrackUtilConan(ConanFile):
+    name = "twitch-libsoundtrackutil"
```

Comment:
```suggestion
    name = "twitchtv-libsoundtrackutil"
```

Follow the same user/company name.

Reply:
@uilianries I don't follow. Could you explain why you suggested prefixing the author name in this case?

Reply:
Okay, you didn't. Still not sure why it was added in the first place

Reply:
@Croydon I suggested because is not a known library and the Wwhuie suggested a twitch, which is not the original author name.